src/ApiConnectors/BaseApiConnector.php

```diff
@@ -5,17 +5,35 @@
 use PhpTwinfield\Enums\Services;
 use PhpTwinfield\Exception;
 use PhpTwinfield\Secure\Connection;
-use PhpTwinfield\Services\BaseService;
 use PhpTwinfield\Services\FinderService;
 use PhpTwinfield\Services\ProcessXmlService;
 
 abstract class BaseApiConnector
 {
+    /**
+     * @var string[]
+     */
+    private const RETRY_REQUEST_EXCEPTION_MESSAGES = [
+        "Error Fetching http headers",
```

Comment:
@mvdpanne I am not sure if the `Error Fetching http headers` should be retried, it may have been received (and processed) by the server. I think you get this during a timeout.

Reply:
Hmm, I looked it up and you're right. It means the connection has been made, but the response does not return in time. So I guess you could always say that the server has processed/is still processing the request.

So yeah, seems right to remove it. What do you think @Daanvm?

Reply:
I'll take care of it, thanks!

Reply:
Yeah, you're right. I'm sorry.